src/sentry/tagstore/base.py

```diff
@@ -211,7 +211,9 @@ def get_tag_value_paginator_for_projects(
         """
         raise NotImplementedError
 
-    def get_group_tag_value_iter(self, project_id, group_id, environment_id, key, callbacks=()):
+    def get_group_tag_value_iter(
+        self, project_id, group_id, environment_id, key, offset, callbacks=()
```

Comment:
Is there any usage of callbacks parameter? Adding the parameter here could break existing callers.

Reply:
I've added the `offset=0` to everywhere this function is used, and the `callbacks` parameter is always named or ignored.

Reply:
Making this parameter required complects all the callsites. If this was an optional parameter with a default value you would have been able to do fewer changes. Given that you've done the work already there is no point in going back, but might be something to keep in mind for next time.

Reply:
I think you're right regardless, I just went back and reverted it. It makes more sense to leave the default.